cmd/prow-job-dispatcher/main.go

```diff
@@ -61,9 +83,257 @@ func (o *options) complete(secrets *sets.String) error {
 		}
 		secrets.Insert(o.prometheusPassword)
 	}
+	if o.maxConcurrency == 0 {
+		o.maxConcurrency = runtime.GOMAXPROCS(0)
+	}
 	return nil
 }
 
+type clusterType int
+
+const (
+	none clusterType = iota
```

Comment:
iotas have the disadvantage that they are unreadable once serialized (`clusterType: 1`) and the main adantage is performance, but the performance of our code here is pretty much irrelvant. Could we change this to a typed string?

Reply:
I will change.
To be fair: is serialization relevant?

Reply:
Removed.
(after refactor, it turned out that we do not need it any more)